src/libs/ajax/User.ts

```diff
@@ -56,7 +56,7 @@ export type TerraUserPreferences = {
 
 export interface OrchestrationUserProfileResponse {
```

Comment:
Is this type used anymore?

Reply:
![image](https://github.com/DataBiosphere/terra-ui/assets/56648914/5b6b1b9c-ebb6-46b2-926b-9a7140329558)
I would like to do this so it is more clear what is going on. Does that seem good?

Reply:
All set now